Gemfile

```diff
@@ -45,6 +45,7 @@ gem 'xml-simple'
 gem 'compact_index', '~> 0.11.0'
 gem 'sprockets-rails'
 gem 'rack-attack'
+gem 'bootsnap', require: false
```

Comment:
Is it necessary out of development group? It seems to not use global environment.

Reply:
no, not at the moment